cli/util/convert.go

```diff
@@ -90,6 +90,11 @@ func NewCommands() []*cli.Command {
 			Name:  "skip-blocks-uploading",
 			Usage: "Skip blocks uploading and upload only index files",
 		},
+		&cli.UintFlag{
+			Name:  "max-retries",
+			Usage: "Maximum number of retries of request",
```

Comment:
Maximum number of NeoFS request retries

Reply:
we retry neo rpc too

Reply:
OK, then just `Maximum number of Neo/NeoFS node request retries`.